scalalib/src/ScalaModule.scala

```diff
@@ -272,9 +293,9 @@ trait ScalaModule extends JavaModule { outer =>
           "-siteroot",
           combinedStaticDir.toNIO.toString
         ),
-        os.walk(compile().classes.path)
-          .filter(_.ext == "tasty")
-          .map(_.toString),
+        docSources()
```

Comment:
Here we need to walk the source directories to filter the files. Same in the other branch.

Reply:
Fixed in https://github.com/com-lihaoyi/mill/pull/1385/commits/e7934292cf2b2bfc781d788e1e3a530d90dc7b46